static/js/popovers.js

```diff
@@ -34,6 +34,26 @@ var list_of_popovers = [];
     }
 }($.fn.popover));
 
+function init_email_clipboard() {
+    var email_el = $('.my_email');
+    var copy_email_clipboard = new Clipboard(email_el[0]);
+
+    // only change the parent element's text back to email
+    // and not overwrite the tooltip.
+    var email_textnode = email_el[0].childNodes[0];
+
+    copy_email_clipboard.on('success', function (e) {
```

Comment:
It looks like we add one of these on-success handlers every time we open a user popover; do these get garbage-collected properly when the popover is hidden?

Reply:
Yep, because its synchronous action i think.